tests/cpp_validation_tests/test_validation.cpp

```diff
@@ -257,20 +327,36 @@ void assert_result(ConstDataset const& result, ConstDataset const& reference_res
                         break;
                     }
                 }
+                // for other _angle attribute, we need to find the magnitue and compare together
+                std::regex const angle_regex("(.*)(_angle)");
```

Comment:
I believe this should allow us to remove the TODO in https://github.com/PowerGridModel/power-grid-model/pull/321/files#diff-729d19abd466fed12f440ac99f0de1c9e94f8008414e7a161cac24b8779f409fR318-R321 , right?

but if I do that, there are 2 failing test cases:
* state_estimation/1os2msr-no-angle-sym-iterative_linear
* state_estimation/dummy-test-sym-sym-iterative_linear

we can also verify this in a follow-up

Reply:
No, that's a different problem. Adding this would not fix that problem.